spec/API_specification/array_api/elementwise_functions.py

```diff
@@ -904,28 +904,58 @@ def less_equal(x1: array, x2: array, /) -> array:
     """
 
 def log(x: array, /) -> array:
-    """
-    Calculates an implementation-dependent approximation to the natural (base ``e``) logarithm, having domain ``[0, +infinity]`` and codomain ``[-infinity, +infinity]``, for each element ``x_i`` of the input array ``x``.
```

Comment:
It looks like the domain/codomain part here is still relevant for real-valued complex numbers, but it seems to be deleted rather than moved. Is that on purpose?